src/core/annotation/PDFAnnotation.ts

```diff
@@ -56,6 +56,11 @@ class PDFAnnotation {
     return AP;
   }
 
+  getNormalAppearance() : PDFRef | PDFDict {
+    const AP = this.ensureAP();
+    return AP.get(PDFName.of('N')) as PDFRef | PDFDict;
```

Comment:
It's best to avoid `as` casts whenever possible (at least in the context of the pdf-lib codebase):
```suggestion
    const N = AP.get(PDFName.of('N'));
    if (N instanceof PDFRef || N instanceof PDFDict) return N;
    throw new Error(`Unexpected N type: ${N?.constructor.name}`);
```